docs-react-native/react-native/docs/installation.md

```diff
@@ -66,7 +66,6 @@ If, however, you're currently compiling your app for SDK version lower than 33,
 
 `npx expo install expo-build-properties`
 
-Then, add `@notifee/react-native` to the list of plugins in your app's Expo config (`app.json` or `app.config.js`):
 ```js
```

Comment:
Be sure to remove the code block as well.

Reply:
in the code block there is no '@notifee/react-native', isn't the addition of 'expo-build-properties' to the plugin list necessary?